lib/containers/runtime.pm

```diff
@@ -131,18 +222,178 @@ sub remove_image {
     $self->_rt_assert_script_run("rmi -f $image_name");
 }
 
+=head2 remove_container
+
+Remove a container from the pool.
+
+=cut
+sub remove_container {
+    my ($self, $container_name) = @_;
+    $self->_rt_assert_script_run("rm -f $container_name");
+}
+
+=head2 check_image_in_host_registry
+
+Returns true if host contains C<img> or false.
+
+=cut
+sub check_image_in_host_registry {
```

Comment:
this is not accurate. The host doesn't contain a registry. The host contain images, but registry is a different thing.
```suggestion
sub check_image_in_host {
```

Reply:
Strongly agree with this.